README.md

```diff
@@ -80,18 +80,18 @@ The workflow to use LCE consists of the following steps:
 
 1. **Run inference**
 
-    LCE uses the [Tensorflow Lite Interpreter](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/guide/inference.md) 
+    LCE uses the [TensorFlow Lite Interpreter](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/guide/inference.md) 
     to perform an inference. In addition to the already available built-in
-    Tensorflow Lite Ops, optimized LCE Ops are registered to the interpreter
+    TensorFlow Lite Ops, optimized LCE Ops are registered to the interpreter
     to execute the Larq specific subgraphs of the model. An example to create
-    and build LCE compatible Tensorflow Lite interpreter in user's applications
+    and build LCE compatible TensorFlow Lite interpreter in user's applications
     is provided [here](./docs/inference.md).
 
 To learn more about using LCE, see [Get started](./docs/get_started.md).
 
 ## Next steps
-- Visit [Get started](./docs/get_started.md) to learn more about how to integrate LCE 
```

Comment:
This PR also removes `/docs/get_started.md` (note the description there is a bit more extensive, please check if we want to keep any of that here).

Reply:
if we remove the `get_started.md` the line 90 needs to be removed as well.